definitions/ext-snowflakewarehouse/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
+type: SNOWFLAKEWH
```

Comment:
Same with this folder name and type.

Reply:
Unless we have a reason to abbreviate i would make these `SNOWFLAKE_WAREHOUSE` IMHO